gradle/ext.gradle

```diff
@@ -33,6 +33,7 @@ gradle.ext {
     library_version_base = '0.2.2'
     library_version = libraryVersionLocalProperties ?: "$library_version_base${System.properties.kaluga_branch_postfix}"
     android_min_sdk_version = 21
+    android_compile_sdk_version = 31
```

Comment:
Android 12 requires changes in activity declarations and permissions. It shall be done in a separate PR

Reply:
Do you mean https://github.com/splendo/kaluga/pull/378 ?

Reply:
nope, that one is related to kotlin. my comment is about Android OS support. there are some changes required so I'm compiling with the most recent sdk 31 (OS Android 12) but targeting 30 (OS Android 11)

Reply:
The PR I link was failing (at the time of writing) because the activity declarations are not updated. It also updates compile SDK so you already have a merge conflict I assume.

So I am asking, is that the follow up PR you refer to? Evidently not, but it's best not to make two PRs to solve the same problem. So please acknowledge the efforts will be combined.